src/components/Books.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from './Book';
+
+const Books = () => {
+  const { books } = useSelector((state) => state.book);
+
+  return (
+    <div>
+      {books.map((item) => (
+        <Book
+          key={item.itemId}
+          title={item.title}
+          author={item.author}
+          category={item.category}
+        />
+      ))}
+    </div>
```

Comment:
- [ ] I cannot remove a book because the `itemId` being used in the Book component is not being passed to it as a prop. To fix this kindly add `itemId` as a prop:

  ```
  itemId={item.itemId}
  ```

  ![image](https://github.com/Dorcas126/Book-store-project/assets/54848363/44eefd03-2002-4385-a233-cefc746c929f)